tests/system/providers/google/cloud/vertex_ai/example_vertex_ai_auto_ml.py

```diff
@@ -0,0 +1,688 @@
+}
+
+IMAGE_DATA_CONFIG = [
+    {
+        "import_schema_uri": schema.dataset.ioformat.image.single_label_classification,
+        "gcs_source": {"uris": [f"gs://{IMAGE_GCS_BUCKET_NAME}/vertex-ai/image-dataset.csv"]},
+    },
+]
+TEXT_DATA_CONFIG = [
+    {
+        "import_schema_uri": schema.dataset.ioformat.text.single_label_classification,
+        "gcs_source": {"uris": [f"gs://{TEXT_GCS_BUCKET_NAME}/vertex-ai/text-dataset.csv"]},
+    },
+]
+VIDEO_DATA_CONFIG = [
+    {
+        "import_schema_uri": schema.dataset.ioformat.video.classification,
+        "gcs_source": {"uris": [f"gs://{VIDEO_GCS_OBJECT_NAME}/vertex-ai/video-dataset.csv"]},
+    },
+]
+
+TEST_TIME_COLUMN = "date"
+TEST_TIME_SERIES_IDENTIFIER_COLUMN = "store_name"
+TEST_TARGET_COLUMN = "sale_dollars"
+
+COLUMN_SPECS = {
+    TEST_TIME_COLUMN: "timestamp",
+    TEST_TARGET_COLUMN: "numeric",
+    "city": "categorical",
+    "zip_code": "categorical",
+    "county": "categorical",
+}
+
+COLUMN_TRANSFORMATIONS = [
+    {"categorical": {"column_name": "Type"}},
+    {"numeric": {"column_name": "Age"}},
+    {"categorical": {"column_name": "Breed1"}},
+    {"categorical": {"column_name": "Color1"}},
+    {"categorical": {"column_name": "Color2"}},
+    {"categorical": {"column_name": "MaturitySize"}},
+    {"categorical": {"column_name": "FurLength"}},
+    {"categorical": {"column_name": "Vaccinated"}},
+    {"categorical": {"column_name": "Sterilized"}},
+    {"categorical": {"column_name": "Health"}},
+    {"numeric": {"column_name": "Fee"}},
+    {"numeric": {"column_name": "PhotoAmt"}},
+]
+
+
+with models.DAG(
+    f"{DAG_ID}_forecasting_training_job",
+    schedule_interval="@once",
+    start_date=datetime(2021, 1, 1),
+    catchup=False,
+    tags=["example", "vertex_ai", "auto_ml"],
+) as forecasting_training_job_dag:
+
+    create_bucket = GCSCreateBucketOperator(
+        task_id="create_bucket",
+        bucket_name=FORECAST_GCS_BUCKET_NAME,
+        storage_class="REGIONAL",
+        location=REGION,
+    )
+
+    unzip_file = BashOperator(
+        task_id="unzip_csv_data_file",
+        bash_command=f"unzip {FORECAST_ZIP_CSV_FILE_LOCAL_PATH} -d /forecast/",
+    )
+
+    upload_files = LocalFilesystemToGCSOperator(
+        task_id="upload_file_to_bucket",
+        src=FORECAST_CSV_FILE_LOCAL_PATH,
+        dst=FORECAST_GCS_OBJECT_NAME,
+        bucket=FORECAST_GCS_BUCKET_NAME,
+    )
+
+    create_forecast_dataset = CreateDatasetOperator(
+        task_id="forecast_dataset",
+        dataset=FORECAST_DATASET,
+        region=REGION,
+        project_id=PROJECT_ID,
+    )
+    forecast_dataset_id = create_forecast_dataset.output['dataset_id']
+
+    # [START how_to_cloud_vertex_ai_create_auto_ml_forecasting_training_job_operator]
+    create_auto_ml_forecasting_training_job = CreateAutoMLForecastingTrainingJobOperator(
+        task_id="auto_ml_forecasting_task",
+        display_name=f"auto-ml-forecasting-{ENV_ID}",
```

Comment:
To not confuse users with "ENV_ID" in the example, we could introduce variable for it (and it's used in two places btw):

```
FORECASTING_DISPLAY_NAME = f"auto-ml-forecasting-{ENV_ID}"
```
Same for  model_display_name=f"auto-ml-forecasting-model-{ENV_ID}"

Reply:
Done